docs/accessControl.md

```diff
@@ -66,13 +66,13 @@ Every course belongs to an **institution** and by default access is only allowed
 
 Each user accesses the PrairieLearn server in a `mode`, as listed below. This can be used to restrict access to assessments based on the current mode. The `mode` setting of an access rule has the following effect:
 
-| Mode        | When is access allowed?                                                                                                                                                                                                          |
-| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `Public`    | Access is normally allowed, but is blocked when the student is signed in for an exam in the Computer-Based Testing Facility (CBTF).                                                                                              |
-| `Exam`      | Access is normally blocked, and is only allowed when the student is signed in to the Computer-Based Testing Facility (CBTF). The `examUuid` should also be specified in the same access rule to limit access to a specific exam. |
-| no mode set | Access is allowed anytime, both during CBTF exams and outside of exams.                                                                                                                                                          |
+| Mode        | When is access allowed?                                                                                                                                                                                                                                                               |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Public`    | Access is normally allowed, but is blocked when the student is checked in on PrairieTest, either in a Computer-Based Testing Facility (CBTF) or in a course-managed session.                                                                                                          |
+| `Exam`      | Access is normally blocked, and is only allowed when the student is checked in on PrairieTest, either in a Computer-Based Testing Facility (CBTF) on in a course-managed session. The `examUuid` should also be specified in the same access rule to limit access to a specific exam. |
+| no mode set | Access is allowed anytime, both during PrairieTest exams and outside of exams.                                                                                                                                                                                                        |
```

Comment:
This is no longer true as of #9743. If `"mode"` is not set in an access rule, it will never be visible when a student is in Exam mode because of PrairieTest.

The subtlety here is that if a student is in Exam mode because they're accessing via a legacy `exam_mode_networks` IP, the old behavior takes effect (a rule without an explicit `mode` will be accessible). I don't know if it's worth putting this in the documentation though, this is virtually never used anymore.

@mwest1066 did I get this all correct? Do you have any better thoughts on how to document this for users?

Reply:
So other than networks, what's the difference between no mode and public mode?

Reply:
Ignoring legacy environments, there is no longer a difference between no mode and public mode. That is, if we have an `exam_uuid` set then we must have `mode = Exam` (it's an error to have anything else). If we don't have an `exam_uuid` then we must have no mode or `mode = Public` and both are treated exactly the same.

@nwalters512 I wonder whether we should change the code (no necessarily immediately) to treat "no mode + `exam_uuid`" as valid and equivalent to the current "`mode = Exam` + `exam_uuid`". This would allow us to remove the documentation about `mode` completely from access rules. We would still retain the code that uses `mode` for legacy environments.

Reply:
@mwest1066 I've been wondering about that as well; it'd be a nice conceptual simplification!

Reply:
TBH, that sounds like a great idea. If for nothing else, it would remove the confusion between `"mode": "Exam"` and `"type": "Exam"` until #3347 is addressed.

Reply:
Docs updated in a5aedf4c8.